ivadomed/main.py

```diff
@@ -463,6 +474,10 @@ def run_main():
     # Get context from configuration file
     path_config_file = args.config
     context = imed_config_manager.ConfigurationManager(path_config_file).get_config()
+    
+    context["command"] = imed_utils.get_command(args)
+    context["log_directory"] = imed_utils.get_log_directory(args)
```

Comment:
If we're refactoring to call it the `path_output`, maybe we should change to:
```
context["path_output"]
```
What do you think?

Reply:
I think that is a great idea actually! Would probably be easier to understand the code base overall by deciding on a uniform. I have noticed different variable names being used to refer to the same thing, so I think it would be worth the effort to refactor

Reply:
Great! Yah makes it easier when they're all the same name :)